.taskcat.yml

```diff
@@ -16,8 +16,6 @@ project:
     RemoteAccessCidr: --override--
     S3TAPConfigBucketName: $[taskcat_random-string]-$[taskcat_current_region]
     EKSClusterName: $[taskcat_random-string]
-    NumberOfNodes: 4
-    MaxNumberOfNodes: 4
```

Comment:
_suggestion_: I think we should also change these in the taskcat template we use in the pipeline, WDYT?

https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/ad1bb93dcc47850c53c7641a67853766b36aba51/ci/qs-test/taskcat.yml#L36-L37

Reply:
yes. Please do so.